code/game/machinery/suit_storage_unit.dm

```diff
@@ -948,7 +948,7 @@
 			occupant.take_organ_damage(0,radiation_level*2 + rand(1,3))
 		if(radiation_level > 1)
 			occupant.take_organ_damage(0,radiation_level + rand(1,3))
-		occupant.radiation += radiation_level*10
+		occupant.apply_effect(radiation_level*10, IRRADIATE)

```

Comment:
I'm surprised this is the only improper use of inflicting radiation damage in the entire codebase.